examples/Demo/Shared/Pages/DateTimes/Examples/DatePickerDefault.razor

```diff
@@ -1,7 +1,7 @@
-<FluentDatePicker Label="Meeting date" @bind-Value="@SelectedValue" ReadOnly />
-<FluentDatePicker AriaLabel="To" @bind-Value="@SelectedValue" />
+﻿<FluentDatePicker Label="Meeting date" @bind-Value="@SelectedValue" ReadOnly />
+<FluentDatePicker AriaLabel="To" @bind-Value="@SelectedValue" Culture="@(new System.Globalization.CultureInfo("fa-IR"))" />
```

Comment:
Please, could you don't change the default sample. But add another Example if you want.